internal/torcx/perform.go

```diff
@@ -183,7 +183,14 @@ func applyImages(applyCfg *ApplyConfig, images Images) error {
 			logrus.WithFields(logFields).WithField("assets", assets.Units).Debug("tmpfiles propagated")
 		}
 
-		// TODO(lucab): evaluate and propagate more units types
+		if len(assets.UdevRules) > 0 {
+			if err := propagateUdevRules(applyCfg, imageRoot, assets.UdevRules); err != nil {
+				failedImages = append(failedImages, im)
+				logrus.WithFields(logFields).WithField("assets", assets.Units).Error("failed to propagate udev rules: ", err)
```

Comment:
I think that should be assets.UdevRules

Reply:
Good catch! Silly copy-pasting from my side :disappointed: